fork.yaml

```diff
@@ -192,7 +193,7 @@ def:
         - title: Node config
           globs:
             - "eth/ethconfig/config.go"
-        - title: Tx gossip disable option
+        - title: Tx gossip disable and netrestrict options
```

Comment:
forkdiff should be updated because trusted peers only flag added. Also the `The tx pool gossip can be restricted to specific IP networks.`